### PR TITLE
feat(android): add API 30 and buildTools 30.0.0

### PR DESCRIPTION
**Pull request by rosariopfernandes**
### Checklist
- [x] I've run `make` from the root directory to see all Dockerfiles are created successfully.
- [x] I've updated the documentation if necessary.

### Motivation and Context

This PR relates to <issue>.

This should add support for Android 11 (API 30) which is now available.

### CODE REVIEW

**Comment by felicianotech** [comment]
Taking a look.

File: android/Dockerfile.m4 [hunk 1]
```diff
@@ -108,7 +108,8 @@ RUN sdkmanager \
   "build-tools;29.0.0" \
   "build-tools;29.0.1" \
   "build-tools;29.0.2" \
-  "build-tools;29.0.3"
+  "build-tools;29.0.3" \
+  "build-tools;30.0.0" \
```
Comment (felicianotech) [inline android/Dockerfile.m4:112]: Hi. This PR is failing due to the trailing forward slash on line 112. Can you remove it please? The line should end after the double quote, no space or trailing slash.

Comment (rosariopfernandes) [reply android/Dockerfile.m4]: Oops! Removed it.

**Comment by mannodermaus** [comment]
Looking forward to seeing this merged into the main branch!

**Pull request merged**